tests/cold/conftest.py

```
@@ -0,0 +1,103 @@
+    if ca_crt_source_provider.ocp_resource is None:
+        raise ValueError("ca_crt_source_provider.ocp_resource is not set")
+
+    providers = {
+        Provider.ProviderType.RHV: OvirtForkliftInventory,
+        Provider.ProviderType.VSPHERE: VsphereForkliftInventory,
+        Provider.ProviderType.OPENSTACK: OpenstackForliftinventory,
+    }
+    provider_class = providers.get(ca_crt_source_provider.type)
+
+    if provider_class is None:
+        raise ValueError(f"Provider {ca_crt_source_provider.type} does not use CA certificates")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)